### ISSUE TITLE
Error for Daniele

### ISSUE THREAD

**Issue by david-daal** [Demigiant/dotween#246]
Hi guys.
Here is error I got. It's asking to share it with you so here you go.
DOTween version : v.1.1.695

How it occured
I got a problem that after reloading the scene DOTween stops working. 
The `OnComplete` functionality was working but the tweening part was not.
So I've added `DOTween.Clear(true)` before calling `SceneManager.LoadScene("");`
That fixed the problem but after reloading I got that error.
Hope as it's promised it will help you making DOTween even more faster :)

```
DOTWEEN :: Error in RemoveActiveTween.totActiveDefaultTweens. It's been taken care of so no problems, but Daniele (DOTween's author) is trying to pinpoint it and it would be awesome if you could reproduce this log in a sample project and send it to him. Or even just write him the complete log that was generated by this message. Fixing this would make DOTween slightly faster. Thanks.
UnityEngine.Debug:Log(Object)
DG.Tweening.Core.Debugger:Log(Object) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/Debugger.cs:22)
DG.Tweening.Core.Debugger:LogRemoveActiveTweenError(String) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/Debugger.cs:70)
DG.Tweening.Core.TweenManager:RemoveActiveTween(Tween) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:941)
DG.Tweening.Core.TweenManager:Despawn(Tween, Boolean) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:217)
DG.Tweening.Core.TweenManager:DespawnActiveTweens(List`1) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:922)
DG.Tweening.Core.TweenManager:Update(UpdateType, Single, Single) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:448)
DG.Tweening.Core.DOTweenComponent:Update() (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/DOTweenComponent.cs:51)
```

UPDATE:
I have same error message after updating to v.1.2.055

**Comment by GGsparta** [comment]
Same problem here. I'd like to clear/complete everything before switching scene, with a code like this:
```
canvas.DOFade(0, .5f).OnComplete(() =>
            {
                DOTween.Clear(true);
                load.allowSceneActivation = true;
            });
```

After the scene reloads I get the messages below:

```
DOTWEEN ► Error in RemoveActiveTween (totActiveDefaultTweens < 0). It's been taken care of so no problems, but Daniele (DOTween's author) is trying to pinpoint it (it's very rare and he can't reproduce it) so it would be awesome if you could reproduce this log in a sample project and send it to him. Or even just write him the complete log that was generated by this message. Fixing this would make DOTween slightly faster. Thanks.
UnityEngine.Debug:LogWarning(Object)
DG.Tweening.Core.Debugger:LogWarning(Object) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/Debugger.cs:35)
DG.Tweening.Core.Debugger:LogRemoveActiveTweenError(String) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/Debugger.cs:94)
DG.Tweening.Core.TweenManager:RemoveActiveTween(Tween) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:1096)
DG.Tweening.Core.TweenManager:Despawn(Tween, Boolean) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:236)
DG.Tweening.Core.TweenManager:DespawnActiveTweens(List`1) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:1074)
DG.Tweening.Core.TweenManager:Update(UpdateType, Single, Single) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:502)
DG.Tweening.Core.DOTweenComponent:Update() (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/DOTweenComponent.cs:73)

DOTWEEN ► Error in RemoveActiveTween (totActiveTweens < 0). It's been taken care of so no problems, but Daniele (DOTween's author) is trying to pinpoint it (it's very rare and he can't reproduce it) so it would be awesome if you could reproduce this log in a sample project and send it to him. Or even just write him the complete log that was generated by this message. Fixing this would make DOTween slightly faster. Thanks.
UnityEngine.Debug:LogWarning(Object)
DG.Tweening.Core.Debugger:LogWarning(Object) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/Debugger.cs:35)
DG.Tweening.Core.Debugger:LogRemoveActiveTweenError(String) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/Debugger.cs:94)
DG.Tweening.Core.TweenManager:RemoveActiveTween(Tween) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:1136)
DG.Tweening.Core.TweenManager:Despawn(Tween, Boolean) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:236)
DG.Tweening.Core.TweenManager:DespawnActiveTweens(List`1) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:1074)
DG.Tweening.Core.TweenManager:Update(UpdateType, Single, Single) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:502)
DG.Tweening.Core.DOTweenComponent:Update() (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/DOTweenComponent.cs:73)

DOTWEEN ► Error in RemoveActiveTween (totActiveTweeners < 0). It's been taken care of so no problems, but Daniele (DOTween's author) is trying to pinpoint it (it's very rare and he can't reproduce it) so it would be awesome if you could reproduce this log in a sample project and send it to him. Or even just write him the complete log that was generated by this message. Fixing this would make DOTween slightly faster. Thanks.
UnityEngine.Debug:LogWarning(Object)
DG.Tweening.Core.Debugger:LogWarning(Object) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/Debugger.cs:35)
DG.Tweening.Core.Debugger:LogRemoveActiveTweenError(String) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/Debugger.cs:94)
DG.Tweening.Core.TweenManager:RemoveActiveTween(Tween) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:1141)
DG.Tweening.Core.TweenManager:Despawn(Tween, Boolean) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:236)
DG.Tweening.Core.TweenManager:DespawnActiveTweens(List`1) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:1074)
DG.Tweening.Core.TweenManager:Update(UpdateType, Single, Single) (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/TweenManager.cs:502)
DG.Tweening.Core.DOTweenComponent:Update() (at D:/DG/_Develop/__UNITY3_CLASSES/_Holoville/__DOTween/_DOTween.Assembly/DOTween/Core/DOTweenComponent.cs:73)
```

**Comment by Demigiant** [comment]
Ahoy! This error was solved in version 1.2.280. Download the latest from the Asset Store or [the website](http://dotween.demigiant.com/download.php) and it should be fixed :)

**Comment by Demigiant** [comment]
P.S. On a secondary note DOTween.Clear(true) exists for very limited cases, mostly debug. You should use Clear() without forcing the complete clearing of everything, so that the DOTweenComponent gameObject stays in the scene and doesn't need to be recreated again at the next usage :)
